src/components/Contribute/Knowledge/AutoFill.ts

```diff
@@ -0,0 +1,262 @@
+    answerValidationError: undefined
+  }
+];
+
+const questionAndAnswerPairs2: QuestionAndAnswerPair[] = [
+  {
+    immutable: false,
+    question: 'What are the characteristics of the Phoenix constellation?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer:
+      'Phoenix is a small constellation bordered by Fornax and Sculptor to the north, Grus to the west, Tucana to the south, touching on the corner of Hydrus to the south, and Eridanus to the east and southeast. The bright star Achernar is nearby.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  },
+  {
+    immutable: false,
+    question: 'When is the phoenix constellation most visible?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer: 'Phoenix is most visible from locations such as Australia and South Africa during late Southern Hemisphere spring.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  },
+  {
+    immutable: false,
+    question: 'What are the Phoenix Constellation boundaries?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer:
+      'The official constellation boundaries for Phoenix, as set by Belgian astronomer Eugène Delporte in 1930, are defined by a polygon of 10 segments.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  }
+];
+
+const questionAndAnswerPairs3: QuestionAndAnswerPair[] = [
+  {
+    immutable: false,
+    question: 'In the Phoenix constellation, how many stars have planets?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer:
+      'In the Phoenix constellation, ten stars have been found to have planets to date, and four planetary systems have been discovered with the SuperWASP project.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  },
+  {
+    immutable: false,
+    question: 'What is HD 142?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer:
+      'HD 142 is a yellow giant that has an apparent magnitude of 5.7, and has a planet (HD 142 b) 1.36 times the mass of Jupiter which orbits every 328 days.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  },
+  {
+    immutable: false,
+    question: 'Are WASP-4 and WASP-5 solar-type yellow stars?',
+    isQuestionValid: ValidatedOptions.success,
+    questionValidationError: undefined,
+    answer:
+      'Yes, WASP-4 and WASP-5 are solar-type yellow stars around 1000 light years distant and of 13th magnitude, each with a single planet larger than Jupiter.',
+    isAnswerValid: ValidatedOptions.success,
+    answerValidationError: undefined
+  }
+];
+
+const questionAndAnswerPairs4: QuestionAndAnswerPair[] = [
```

Comment:
We need one more qna pair to satisfy the knowledge schema constraints. GUI also shows only 4 seed examples, so it doesn't allow to submit the knowledge form.

Reply:
I followed the example case here https://pages.github.ibm.com/research-uki/newsbeat/2024/august.html#/ADMM.
I will add another on my end. Should I add an issue to the Taxonomy repo to get this updated to 5 examples?

Reply:
@aevo98765 can't access the url , but yes, i think we should report the issue to get that fixed.